plugin-server/src/main/ingestion-queues/kafka-queue.ts

```diff
@@ -180,6 +189,23 @@ export class KafkaQueue {
             .some(({ topic, partitions }) => topic === targetTopic && (!partition || partitions.includes(partition)))
     }
 
+    async isHealthy(): Promise<boolean> {
```

Comment:
If we weren't rushing this - all plugin-server code should have proper tests, including this.